token-allocation/README.md

```diff
@@ -0,0 +1,78 @@
+# Chainlink Token Allocation Price Adapter
+
+The adapter calculates the total value in the currency selected for the selected tokens
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+- `DATA_PROVIDER`: Data provider to use. Some of them require an `API_KEY`(K). Options available:
+    - `amberdata` (K)
+    - `coinapi`(K)
+    - `coingecko`
+    - `coinmarketcap`(K)
+    - `coinpaprika`
+    - `cryptocompare`
+    - `kaiko` (K)
+    - `nomics`(K)
+- `API_KEY`: For those data providers who need an api key
+- `DEFAULT_CURRENCY` (Optional): Currency that the price will be fetched by default. `USD` used by default
+
+
+## Input Params
+
+- `components`: Array of the token symbols. 
+- `units`: Array of balances in wei of each token.
+- `currency` (optional). Currency we want the price on. `DEFAULT_CURRENCY` by default
+
+`units[n]` would correspond to the `components[n]` balance
+
+```json
+{
+  "jobID": "1",
+  "data": {
+    "components": [
+      "DAI",
+      "USDC",
+      "USDT"
+    ],
+    "units": [
+      "10000000000000000000",
+      "10000000000000000000",
+      "10000000000000000000"
```

Comment:
Does this match with the output in this readme?
And why does units change from wei to "Ether"?

Reply:
- It doesn't match you're right. Will fix it. 
- It would make sense to be the same units actually, either both 'wei' or both 'ether'. Wei I'd say makes more sense as it's more precise, but not sure.

Reply:
I agree. On the last point, also consider that "ether" more sense normally, if you're not explicitly aware that this adapter takes "wei".

How much precision do we usually need with an adapter like this? I'm guessing there's a reason why you picked wei to begin with 😅

Reply:
Haha there are some adapters that query the units from contracts directly, so they give it in wei. But they can be converted to "ether" of course 😄

Reply:
Alright, sounds like the easiest thing to do is just give it in wei then.